ql/pom.xml

```diff
@@ -863,6 +863,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-hplsql</artifactId>
```

Comment:
I think it would be better to avoid pulling in hplsql under ql - there is only a console class and some minor other stuff - I think this could be dodged with a few interfaces.

Reply:
I removed this dependency.